widget/discrepancy-note/dn-widget.js

```diff
@@ -1599,6 +1599,13 @@ class Comment extends Widget {
         );
     }
 
+    _decodeHtml(str) {
+        if (str == null) {
+            return "";
+        }
+        return str.replaceAll("&lt;", "<").replaceAll("&gt;", ">").replaceAll("&#39;", "'").replaceAll("&quot;", "\"").replaceAll("&amp;", "&");
+    }
+
```

Comment:
Did you compare the solution with `_encodeHtml` and chose string replacements with `replaceAll` for a reason? I am just curious. It is fine though.

Reply:
No, I just think replaceAll is easier to use, but after seeing your commend I tried searching and it looks like replace is faster than replaceAll so I updated the code